src/utils/debounce.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Returns a function that will not be triggered after right after a call, waiting ${wait} seconds
+ * to be able to invoke it again. If ${immediate}, it will call again in the leading edge, instead of trailing.
+ * It's a adapted version of {@link http://underscorejs.org/#debounce}
```

Comment:
Oof, that's tough for such a small thing, thanks for checking. Then I guess we use the custom debounce for size reasons and further investigate how we can shrink the module size :grin: